content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -262,6 +262,22 @@ This annotation is part of the Kubernetes Resource Model (KRM) Functions Specifi
 which is used by Kustomize and similar third-party tools.
 For example, Kustomize removes objects with this annotation from its final build output.
 
+### config.kubernetes.io/depends-on
+
+Type: Annotation
+
+Example: `config.kubernetes.io/depends-on: "OBJECT_REFERENCE"` 
+
+Used on: All objects
+
+This annotation is used on the target object to be modified to configure [Depends-on](https://kpt.dev/reference/annotations/depends-on/). This annotation specifies one or more resource dependencies.
+The annotation value accepts a list of resource references, delimited by commas.
+
+**Resource reference**
+- **Group**: This represents the API group of the resource (e.g., apps, v1, extensions).
+- **Kind**: This indicates the type of resource (e.g., Pod, Service, Deployment).
+- **Name**: The name is the unique identifier of the specific resource instance.
```

Comment:
```suggestion
For eg. If `config.kubernetes.io/depends-on: /namespaces/default/Pod/pod-a` annotation is added to `pod-b`,  it identifies that pod-a must be successfully applied prior to pod-a actuation.
```
I think the link will explain what are group, kind, and name. Also, they weren't mentioned in the example and seem like popped up out of nowhere.

Reply:
Please be noted the eg is already available on the linked page. However, we can add it in place of the Example section. If it seems repetitive I think the present Example section works fine as well, but less informative. Just feel like the explanation helps understanding annotation better.
WDYT, @sftim ?  need a second opinion.

Reply:
Hey @Ritikaa96, I have made the changes could you please give it a look